Orchestrator.py

```diff
@@ -140,17 +144,137 @@ def receive(self,frame):
             duration = frame['movementDuration'],
         )
 
+        # update explored cells
+        self.obstacleCells  += [self._xy2hCell(newX, newY)]
+        cellsTraversed       = self._computeCellsTraversed(dotbot['x'], dotbot['y'], newX, newY)
+        self.exploredCells  += [c for c in cellsTraversed if c not in self.obstacleCells]
+
+        # if a cell is obstacle, remove from open cells
+        try:
```

Comment:
similarly, this logic should not be there. Plus, doing a silent `pass` on errors is a big no-no